core/txpool/bundlepool/bundlepool.go

```diff
@@ -175,8 +175,10 @@ func (p *BundlePool) AddBundle(bundle *types.Bundle, originBundle *types.SendBun
 	p.mu.Lock()
 	defer p.mu.Unlock()
 
-	if price.Cmp(p.minimalBundleGasPrice()) <= 0 && p.slots+numSlots(bundle) > p.config.GlobalSlots {
-		return ErrBundleGasPriceLow
+	if p.slots+numSlots(bundle) > p.config.GlobalSlots {
```

Comment:
I think this section should be placed after line 186. After this bundle successfully drops some bundles, it is strange that it does not enter the bundle pool because it already exists.

Reply:
Make sense. Fixed.